# Message with information about the snapCircuitsPart

int32     id
string label
string  name
location loc

